content/docs/getting-started.md

```diff
@@ -34,6 +34,7 @@ The daemon also uses a configuration file located in `/etc/containerd/config.tom
 A sample configuration file looks like this:
 
 ```toml
```

Comment:
Better to just copypaste the result of `containerd config default`

Reply:
@AkihiroSuda Done. Please take a look and let me know if anything else if required. I also added systemctl commands to start and enable the containerd service.